config/c-code/tests-cache.j2

```diff
@@ -4,9 +4,7 @@
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
-{% if with_future_python %}
           allow-prereleases: true
-{% endif %}
```

Comment:
This means that e. g. `AccessControl` for all Python versions would be tested against the `dev` release for the future Python version instead of the latest stable release. I am not sure whether we always want this.

Reply:
The setting means it will only choose a prerelease if no stable release is available. It is required here because Python 3.13 has no final release. For all other Python versions it has no adverse effect.

Reply:
Cool, I did not know about this. Thank you for the clarification.